sqlmesh/core/macros.py

```diff
@@ -648,6 +648,48 @@ def eval_(evaluator: MacroEvaluator, condition: exp.Condition) -> t.Any:
     return evaluator.eval_expression(condition)
 
 
+@macro()
+def star(
+    evaluator: MacroEvaluator,
+    relation: exp.Table,
+    alias: t.Optional[exp.Identifier] = None,
+    except_: t.Optional[exp.Array] = None,
+    prefix: t.Optional[exp.Literal] = None,
+    suffix: t.Optional[exp.Literal] = None,
+    quote_identifiers: exp.Boolean = exp.Boolean(this=True),
+) -> t.List[exp.Expression]:
+    if alias and not isinstance(alias, exp.Identifier):
```

Comment:
you don't have control over this being an Identifier, it's going to be a column type

Reply:
I see, good to know! Its a little ambiguous to me why this is the case. But if it is logical and concrete, I will lean into this knowledge.